src/conda/devcontainer-feature.json

```
@@ -22,6 +22,7 @@
     },
     "containerEnv": {
         "CONDA_DIR": "/opt/conda",
+        "CONDA_SCRIPT":"${CONDA_DIR}/etc/profile.d/conda.sh",
         "PATH": "${PATH}:${CONDA_DIR}/bin:"
```

```suggestion
        "PATH": "${PATH}:${CONDA_DIR}/bin"
```

I know you didn't change this, but @jungaretti's eagle eyes caught this.